src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -2546,18 +2545,16 @@ void GenericThreadStackManagerImpl_OpenThread<ImplClass>::OnDnsBrowseResult(otEr
 
         VerifyOrExit(error == CHIP_NO_ERROR, );
 
-        DnsResult * dnsResult = Platform::New<DnsResult>(aContext, MapOpenThreadError(aError));
+        DnsResult * dnsResult = Platform::New<DnsResult>(aContext, CHIP_NO_ERROR);
```

Comment:
Why this change?

Reply:
At this line number there is no way that the `aError` indicates an error. It is verified on line 2530. I though that it will easier to read the code if there will be no error conversion but simply `CHIP_NO_ERROR` to satisfy the `DnsResult` API - it's easier to follow that `DispatchBrowse` is called with success result code.

Reply:
>  It is verified on line 2530

Ah, perfect.  That was outside the diff context.  :)